### PR TITLE
status, status_p

### PR DESCRIPTION
**Pull request by hwjsnc**

### CODE REVIEW

File: schemas/cs3/tbl/status.ksy [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  - id: drop_id_1
+    type: u2
+  - id: drop_chance_1
+    type: u1
+  - id: drop_id_2
+    type: u2
+  - id: drop_chance_2
+    type: u1
+
+  - id: unknown_float_8
```
Comment (nnguyen259) [inline schemas/cs3/tbl/status.ksy]: Could be a wild goose chase here, but what if these 2 are drop chance growth?

File: schemas/cs3/tbl/status.ksy [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  Present in t_mons.tbl
+seq:
+  - id: script
+    type: strz
+  - id: texture
+    type: strz
+  - id: model
+    type: strz
+
+  - id: unknown_float_1
```
Comment (nnguyen259) [inline schemas/cs3/tbl/status.ksy:30]: Some of these are related to size, such as model size and hitbox size, but I don't remember which is which

File: schemas/cs3/tbl/status.ksy (hunk 1 above)
Comment (hwjsnc) [reply schemas/cs3/tbl/status.ksy]: They are related to the random scaling of monster models and stats somehow, but I don't understand exactly how. It's not a straight multiplier anyway. (If you want to test, note that the values are stored in the save file and you need to leave and re-enter the map for them to change.)

File: schemas/components/cs3/difficulty.ksy [hunk 3]
```diff
@@ -0,0 +1,14 @@
+meta:
+  id: difficulty
+  endian: le
+seq:
+  - id: difficulty_id
+    type: u2
+    enum: difficulties
+enums:
+  difficulties:
```
Comment (nnguyen259) [inline schemas/components/cs3/difficulty.ksy:9]: Should this be in common or just cs3? I honestly don't remember the difficulty id for cs1-2 and not sure about haji

Comment (hwjsnc) [reply schemas/components/cs3/difficulty.ksy:9]: CS1/2 don't have a Very Easy difficulty. CS4 has the same ones as CS3 so I assume the values would be shared too (though I have no way to verify). It's probably easiest to keep it separate.

**Comment by hwjsnc** [comment]
Big update with more info on the random stat variation, some fixed typos, and tests for everything (unless I missed something). Should be good to merge now, possible mistakes excepted.

**Pull request merged**